PaddleAudio/README.md

```diff
@@ -22,14 +22,37 @@ pip install .
 
 ## Quick start
 ### Audio loading and feature extraction
+``` python
+import paddleaudio
+s,r = paddleaudio.load(f)
```

Comment:
It seems use `s, r` as return results is not that clear.
BTW, `s,r` -> `s, r`, need a space between variables.

Reply:
changed to wav, sample_rate = paddleaudio.load(f)

Reply:
fixed